docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -129,9 +129,9 @@ The order in which naming rules are defined in an EditorConfig file doesn't matt
 
 If you don't specify any custom naming rules, the following default styles are used:
 
-- For classes, structs, enumerations, properties, and events with `public`, `private`, `internal`, `protected`, or `protected_internal` accessibility, the default naming style is Pascal case.
+- For classes, structs, enumerations, properties, methods, and events with `public`, `private`, `internal`, `protected`, `private_protected`, or `protected_internal` accessibility, the default naming style is Pascal case.
```

Comment:
```suggestion
- For classes, structs, enumerations, properties, methods, and events with any accessibility, the default naming style is Pascal case.
```

Reply:
I like @gewarren 's suggestion. When I read this, you've enumerated *all* the possible access levels. It's simpler to just say that.
 (Historically, I bet that at one time `private` used a different standard).